shared/lib_battery_voltage.cpp

```diff
@@ -342,7 +342,7 @@ void voltage_dynamic_t::parameter_compute() {
 }
 
 void voltage_dynamic_t::set_initial_SOC(double init_soc) {
-    updateVoltage(init_soc * 0.01 * params->dynamic.Qfull, params->dynamic.Qfull, 0, 25, params->dt_hr);
+    updateVoltage(init_soc * 0.01 * params->dynamic.Qfull * params->num_strings, params->dynamic.Qfull * params->num_strings, 0, 25, params->dt_hr);
```

Comment:
Please add comments to the header file with the final units convention.

I did a "find all references" check for this function - should lib_battery_test be updated as well?

Reply:
See latest commit to see if comments are sufficient. 

On the second point in lib_battery_test the inputs to updateVoltage come from the capacity model functions that return correct system capacity inputs (confirmed in debugger).